src/js/api/config.js

```diff
@@ -76,6 +76,15 @@ function _normalizeSize(val) {
     return val;
 }
 
+
+function _adjustDefaultBwEstimate(estimate) {
+    if (_isValidNumber(parseFloat(estimate))) {
```

Comment:
You are doing `parseFloat` in the if statement, but not for the value returned. Maybe you should do it before calling this function `_adjustDefaultBwEstimate(parseFloat(config.defaultBandwidthEstimate))`?

Also @johnBartos